modules/ob-installing-buildah-strategy.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-buildah-strategy_{context}"]
+= Installing buildah strategy
+
+You can install the `buildah` cluster build strategy, which does not require a daemon to run and is available for use by unprivileged users.
+
+.Prerequisites
+
+* You have installed the `oc` CLI.
+
+.Procedure
+
+* Run the following command to install the `buildah` strategy:
++
+[source,terminal]
+----
+oc apply -f https://raw.githubusercontent.com/shipwright-io/build/main/samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml
```

Comment:
Need to hold here. I am still awaiting a confirmation from team, about which strategies will be useful here in the context of OpenShift.

Reply:
sure, we can wait for the confirmation.

Reply:
Got that confirmed. We can proceed as per https://github.com/openshift/openshift-docs/pull/63282#discussion_r1290256853